DAQ_FW/lib/Log/Log.cpp

```diff
@@ -1,5 +1,11 @@
 #include "Log.h"
 
+// Output Log Template:
+// TIME [LOG_LEVEL] : LOG INFO STRING
+// 17:20:10:38 [Info] : ArduinoLog System Start Success
+
+// uses printf-like structure, ie: Error(format string, data_variables)
+
 namespace Logger
```

Comment:
I think this might be a good place to use `using namespace Logger;` so you don't have to tab everything out. 

Also, as a good rule of thumb, anything public and accessible to the user should be displayed first in a file. If I want to see the implementation of a function I just called, I'd have to scroll through a bunch of "private" static functions that I have no knowledge of anyways to get to the stuff I care about. See how the VCU's Logger file is structured in terms of public/private variables and functions. Functions in the c/cpp file are also in the same order as they appear in the header file.

https://github.com/sfuphantom/vcu-fw/blob/develop/VCU/Phantom/tasks/source/task_logger.c